planemo/io.py

```diff
@@ -182,6 +182,7 @@ def kill_pid_file(pid_file):
 
     pid = int(open(pid_file, "r").read())
     kill_posix(pid)
+    os.unlink(pid_file)
```

Comment:
This should probably be in a `try/except Exception: pass`, since the killed process can remove the pid file before exiting, in which case  `os.remove` would raise `OSError` or `FileNotFoundError`.

Reply:
Good point, thanks!